consumer-rest-core/src/test/scala/za/co/absa/spline/consumer/rest/controller/ExecutionPlansControllerTest.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2022 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package za.co.absa.spline.consumer.rest.controller
+
+import org.mockito.ArgumentMatchers.{any, eq => eqTo}
+import org.mockito.Mockito.when
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.matchers.should.Matchers
+import org.scalatestplus.mockito.MockitoSugar
+import za.co.absa.spline.consumer.service.model._
+import za.co.absa.spline.consumer.service.repo.ExecutionPlanRepository
+
+import java.util.UUID
+import scala.concurrent.Future
+
+class ExecutionPlansControllerTest extends org.scalatest.flatspec.AnyFlatSpec with Matchers with MockitoSugar with ScalaFutures {
+  import ExecutionPlansControllerTest._
+
+  behavior of "ExecutionPlansControllerTest"
```

Comment:
I think the idea behind `behavior of` is to describe what object you are testing and then just use it to refer to it.

like here:
https://www.scalatest.org/scaladoc/1.8/org/scalatest/FlatSpec.html

It could also be used to sub-dived the test class into several parts each testing different thing.